### PR TITLE
fix(core): ai subscribe button is not show in ai usage page

### PR DESCRIPTION
**Pull request by CatsJuice**

### CODE REVIEW

**Comment by CatsJuice** [comment]
* **#8869** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8869?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8869?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `canary`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/frontend/core/src/desktop/dialogs/setting/general-setting/plans/ai/actions/subscribe.tsx [hunk 1]
```diff
@@ -74,7 +74,12 @@ export const AISubscribe = ({
       onBeforeCheckout={onBeforeCheckout}
       checkoutOptions={checkoutOptions}
       renderer={props => (
-        <Button variant="primary" {...props} {...btnProps}>
+        <Button
+          variant="primary"
+          {...props}
+          {...btnProps}
+          data-testid="ai-subscribe-button"
```
Comment (forehalo) [inline packages/frontend/core/src/desktop/dialogs/setting/general-setting/plans/ai/actions/subscribe.tsx:81]: no test?

Comment (CatsJuice) [reply packages/frontend/core/src/desktop/dialogs/setting/general-setting/plans/ai/actions/subscribe.tsx:81]: Added a test, but seems pricing request not working in CI? So I removed.

Comment (forehalo) [reply packages/frontend/core/src/desktop/dialogs/setting/general-setting/plans/ai/actions/subscribe.tsx:81]: payment service is not enabled in CI, I will configure it later

**Review by forehalo — APPROVED** [review]

**Pull request merged**